docker/check-cpu.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
```

Comment:
It would be good to add few lines of comments here to explain the goal of this file.

Reply:
https://github.com/alicevision/AliceVision/commit/5c2c6e25855bb8f929864a41ffa9a4635ad28e18

contains the commit where I have given an exposition of the utility of the file.

Reply:
Thanks!